Longest_Palindromic_Subsequence/Longest_Palindromic_Subsequence.java

```diff
@@ -32,13 +31,14 @@ public static String print_lps(int[][] lps_dp, int len, String str, String rev)
     }
 
     /*
-        We use Dynamic Programming approach to solve this problem.
-        A 2D matrix M of size n*n is created where n is the length of the given string.
-        Here the coulmn j denotes the char S[j] of the string while row i denotes char R[i], where
-        R is the reverse of the string S.
-        Thus, M[i][j] denotes the max length of longest common subsequence between the string 
-        S[0...j] and R[0..i]. Hence M[n][n] will compute the longest common subsequence of S[0..n]       and R[0..n], which is the required answer.
-    */
+		We use Dynamic Programming approach to solve this problem.
```

Comment:
Use 4 space indentation in this comment also

Reply:
Done. Review please.

Reply:
@SakshiShreya